charts/ai-software-templates/chatbot/templates/application-gitops-job.yaml

```diff
@@ -12,7 +12,118 @@ spec:
     spec:
       containers:
         - name: repo-creator
-          image: quay.io/redhat-ai-dev/helm-chart-application-gitops:latest
+          command:
+            - /bin/bash
+            - -c
+            - |
+              #!/usr/bin/env bash
+              set -o nounset
+              set -o pipefail
+
+              setup_data() {
+                 gh repo clone "$GITHUB_SOURCE_REPO" app-source -- --no-tags --branch="$GITHUB_DEFAULT_BRANCH" --single-branch
+                 gh repo clone "$GITHUB_TETKON_SOURCE_REPO" tekton-source -- --no-tags --single-branch
+                 gh repo clone "$GH_REPO" "$APP_NAME" -- --no-tags --branch="$GITHUB_DEFAULT_BRANCH" --single-branch
```

Comment:
I don't know if it would make things any easier or faster, but we could also wget the source and untar it instead of doing a git clone, that way we don't even have to deal with the fact that they would have a .git directory in them. Then we could just copy things where they need to go?

Reply:
ooh good call ..... https://askubuntu.com/questions/939830/how-to-download-a-github-repo-as-zip-using-command-line

I'll try it and see

Reply:
yeah being able to pick the destination directory with gh clone made it easier to cd into the directory.

with wget it forces me to learn the repo name, which means I have to parse the env vars.

so I'm going to punt on wget @coreydaley .... good idea to investigate though